### ISSUE TITLE
Bug when I try to use the new Presto Driver

### ISSUE THREAD

**Issue by rubenssoto** [metabase/metabase#17199]
Hello,

Im trying to test the new presto driver but I got some errors.

I use this docker image:
docker run -it -p 3000:3000 metabase/metabase-head

Error:
2021-07-25 01:11:40,339 ERROR api.database :: Não posso conectar ao banco de dados
java.lang.Exception: Unrecognized connection property 'engine'

Configuration:
<img width="1680" alt="Captura de Tela 2021-07-24 às 22 13 52" src="https://user-images.githubusercontent.com/36298331/126884749-b438e658-247d-42ef-98bd-a6b50513d896.png">

Could you help me?

Thank you so much.

**Comment by flamber** [comment]
Hi @rubenssoto 
Which Presto type and version are you connecting to?

**Comment by rubenssoto** [comment]
Hello,

PrestoDB, version 0.245.

**Comment by rubenssoto** [comment]
Hello @flamber could you help me with this?

Im so sorry for disturbing you, but to choose metabase to our stack presto is primordial.

Thank you so much

**Comment by flamber** [comment]
@jeff303 

I can reproduce this issue - probably didn't even need the actual DB, but just got it for full testing:
```
 docker run --rm -d -p 8080:8080 --name prestodb ahanaio/prestodb-sandbox:0.246
 docker run --rm -d -p 3000:3000 --name metabase-head metabase/metabase-head
```

It's sending the `presto-jdbc` engine name, but I'm guessing it's not correctly registered - because of the proxy stuff?
`{"engine":"presto-jdbc","name":"prestodb","details":{"host":"172.17.0.1","port":8080,"catalog":"hive","schema":null,"user":"hadoop","password":null,"ssl":false,"additional-options":null},"auto_run_queries":false,"is_full_sync":true,"schedules":{}}`

I'm seeing also seeing a different error. When I enable the "This is a large database...", then it errors with `let-user-control-scheduling` (and I have to do a browser refresh to try anything else).

And the field below the dbname is a schema, but it's missing a title - wouldn't that fix 16973?

**Comment by rubenssoto** [comment]
@flamber Does schema field will be mandatory? Because would be good to have all schemas in the same datasource.

**Comment by flamber** [comment]
@rubenssoto No, it's not set as required.
https://github.com/metabase/metabase/blob/80b46b1f75382ca3e806334f6fc01f67ed484d68/modules/drivers/presto-jdbc/resources/metabase-plugin.yaml#L22-L23

**Comment by jeff303** [comment]
Should be fixed by #17265, once merged.  Thanks for reporting!

**Comment by rubenssoto** [comment]
Thank you for fixing the issue, I will test asap

**Comment by rubenssoto** [comment]
Hello,

Thank you so much for the fix.

I could connect in my presto service, but I had problems with one metabase option:

<img width="1680" alt="Captura de Tela 2021-08-02 às 19 11 34" src="https://user-images.githubusercontent.com/36298331/127930299-095a24af-60fa-4859-95f6-e776186862bb.png">


Unrecognized connection property 'let-user-control-scheduling'

**Comment by jeff303** [comment]
Reopening as per last comment, which should be fixed via #17293.
